### DISCUSSION TITLE
https://angular.love/implementing-reusable-and-reactive-forms-in-angular

### DISCUSSION THREAD

**Discussion by giscus** [HouseOfAngular/angular-love#367] [General]
# https://angular.love/implementing-reusable-and-reactive-forms-in-angular

Angular.love - a place for all Angular enthusiasts created to inspire and educate.

https://angular.love/implementing-reusable-and-reactive-forms-in-angular

**Comment by raven1152** [comment]
love this article, but wish examples were not dependent on nebular (as near as i can tell).  i'm working in angular 19, so i am not able to run the code as is because the latest nebular wants angular 18.  i opted to pull out the nebular markup and replace it with good old html and css, and it works, it just took a lot of extra time to work the examples.
